docs/glossary.rst

```diff
@@ -31,6 +31,12 @@ Astropy Glossary
       ``Quantity("180d")`` is 180 **days**, while ``Angle("180d")`` is 180
       **degrees** -- so check the string parses as intended for ``Quantity``.
 
+   ['physical type']
+       The physical type of a quantity can be annotated in square brackets
+       following a `~astropy.units.Quantity` (or similar :term:`quantity-like`).
+
+       For example, `` distance : quantity-like ['length'] ``
```

Comment:
Hm, this didn't render properly... is it the spaces around the backticks? https://astropy--11595.org.readthedocs.build/en/11595/glossary.html#term-physical-type

```suggestion
       For example, ``distance : quantity-like ['length']``
```

Reply:
Cool - feel free to push a fix with `[ci skip]`

Reply:
Yup. That was it.

Reply:
🤦